src/utils.ts

```diff
@@ -362,10 +347,18 @@ export const overrideFormMethod = (
  */
 export const upgradeInternals = (ref: FormAssociatedCustomElement) => {
```

Comment:
nit: Can we add an explicit return type here?

Honestly if I were writing this again I'd do that everywhere, but while we're changing this function I think it would be better.